src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -24,6 +24,7 @@ namespace System.Net.Http.Functional.Tests
     public class HttpClientHandlerTest
     {
         readonly ITestOutputHelper _output;
+        private const string ExpectedContent = "Test contest";
```

Comment:
Maybe this was just David's way of challenging the code to get it right, a "contest" if you will :wink: